code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -115,6 +115,13 @@
 	can_hold = list(/obj/item/weapon/reagent_containers/food/snacks/grown,/obj/item/seeds,/obj/item/weapon/grown)
 	burn_state = FLAMMABLE
 
+/obj/item/weapon/storage/bag/plants/can_be_inserted(obj/item/W, stop_messages = 0, mob/living/user)
+	if(istype(W, /obj/item/weapon/reagent_containers/food/snacks/grown/cherry_bomb))

```

Comment:
I wasn't meaning for it to be like that oranges, I meant like, just don't go down that route.
also a proc is overkill.